src/synthesiser/Synthesiser.h

```diff
@@ -17,6 +17,7 @@
 #pragma once
 
 #include "RecordTable.h"
+#include "ram/Expression.h"
```

Comment:
This include does not seem to be used in this file. Remove it if it is not used.